CHANGELOG.md

```diff
@@ -1,6 +1,18 @@
 ## Master
 
-## 2.0.0 (2018-12-21)
+## 2.0.1 (2019-01-08)
```

Comment:
Should we go ahead and update this to today's date?

Reply:
yes